config/clusters/dandi/common.values.yaml

```diff
@@ -0,0 +1,146 @@
+nfs:
+  enabled: true
+  pv:
+    enabled: true
+    # from https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-nfs-mount-settings.html
+    mountOptions:
+      - rsize=1048576
+      - wsize=1048576
+      - timeo=600
+      - soft # We pick soft over hard, so NFS lockups don't lead to hung processes
+      - retrans=2
+      - noresvport
+    serverIP: fs-0bf8c8fce5ca8695f.efs.us-east-2.amazonaws.com
+    baseShareName: /
+jupyterhub:
+  hub:
+    config:
+      JupyterHub:
+        authenticator_class: github
+      Authenticator:
+        enable_auth_state: true
+        admin_users:
+          - asdf
```

Comment:
```suggestion
```

Reply:
I *think* this still needs to be removed

Reply:
Ah, I think I've overwritten the commit 🤦🏼‍♀️  Thank you, again :D